src/github/webhook.py

```diff
@@ -130,6 +130,7 @@ def _handle_status_webhook(payload: dict) -> HttpResponse:
         return HttpResponse("200")
 
     with dynamodb_lock(pull_request.id()):
+        pull_request = graphql_client.get_pull_request(pull_request.id())
```

Comment:
Why pull the pull_request a second time here?

Reply:
There's no way for me to get the `CheckRun.isRequired` field without either a PR number of a PR id neither of which is available in the webhook payload. Here, I'm remaking the request to get a fully formed PR object with checkrun information.